src/main/java/org/jabref/gui/util/ControlHelper.java

```diff
@@ -119,6 +119,7 @@ public static String truncateString(String text, int maxCharacters, String ellip
         if (ellipsisString == null) {
             ellipsisString = "";
         }
+        text = text.replace("_", "__");
```

Comment:
a) this will replace every occurend of an underscore. We only need to do ths if the first char is a underscore
b) This is not the correct position to do the replacement  as the method should not do any magic as otherwise specificed (e.g. truncate)

=> Better create a new method for replaceFirstUnderscore and call this where needed